backend/tracim_backend/config.py

```diff
@@ -392,7 +392,7 @@ def _load_global_config(self) -> None:
         self.WEBSITE__TITLE = self.get_raw_config("website.title", "Tracim")
         self.WEB__NOTIFICATIONS__EXCLUDED = self.get_raw_config(
             "web.notifications.excluded",
-            "user.created,user.modified,user.deleted,user.undeleted,workspace.modified,workspace.deleted,workspace.undeleted,workspace_member.modified,content.modified",
+            "user.created,user.modified,user.deleted,user.undeleted,workspace.modified,workspace.deleted,workspace.undeleted,workspace_member.modified,content.modified,reaction",
         )
```

Comment:
Wouldn't have it been more explicit with `reaction.*`?

Reply:
Also we probably should write a short notice in the changelog to advise adding value if the setting was changed in `development.ini`.

Reply:
We can go for reaction.* :-)